password.c

```diff
@@ -132,6 +132,8 @@ void swaylock_handle_key(struct swaylock_state *state,
 	case XKB_KEY_Alt_R:
 	case XKB_KEY_Super_L:
 	case XKB_KEY_Super_R:
+	case XKB_KEY_ISO_Prev_Group:
```

Comment:
This probably doesn't belong to this pull request.

Reply:
Well, it's certainly more related than [3db0b6](https://github.com/swaywm/swaylock/pull/89/commits/3db0b6). The point of resizing the buffer is to properly show the keymap name, which is usually changed with those keys.

Reply:
When the keymap changes, a `wl_keyboard.modifier` event is triggered. This is where we should re-render, not here.

I see that you've split this PR into two commits, which seems like a good idea to me.

Reply:
Doesn't seem to be the case. You can try it out with and without that commit and see for yourself.

Reply:
I do get a `wl_keyboard.modifiers` event when switching layouts. Instead of handling these keys, please damage the state in `keyboard_modifiers`.

Reply:
Updated. @emersion, thank you for your patience.